mocks/service/index.js

```diff
@@ -1,3 +1,4 @@
 require('babel-polyfill');
 require('babel-register');
+require('babel-polyfill');
```

Comment:
We need to require `babel-polyfill` twice?

Or is it a merge issue - I remember adding this line too when implementing GH repo listing API, so I guess we both did it and it ended up twice in the file.